packs/data/feat-effects.db/effect-pistoleros-challenge-legendary.json

```diff
@@ -0,0 +1,45 @@
+{
+    "_id": "1q4VIzG7mtYykEsf",
+    "img": "systems/pf2e/icons/equipment/weapons/flintlock-pistol.webp",
+    "name": "Effect: Pistolero's Challenge (Legendary)",
```

Comment:
Consolidate all three effects instead of making 3 effects.

Reply:
We can at the very least ask which skill was used in a ChoiceSet and then detect it.

Reply:
Can I evaluate a proficiency of the owner of the feat when applying an effect to another actor? Cause that's what would be needed here, right?

Reply:
I could use a Choice Set to select Proficiency instead. Not as pretty though.